PixivBatchHandler.py

```diff
@@ -201,7 +183,7 @@ def process_batch_job(caller: PixivUtil2, batch_file=None):
 
     if os.path.exists(batch_file):
         jobs_file = open(_default_batch_filename, encoding="utf-8")
-        jobs = demjson.decode(jobs_file.read())
+        jobs = demjson.decode(jobs_file.read()) #is there a reason to use demjson over json?
```

Comment:
I remember that pixiv has a json that out-of-spec, where the key is an integer, not a string.

For this instance, I think we can use built-in json lib